pmlc/dialect/stripe/ops.td

```diff
@@ -158,3 +157,11 @@ def TerminateOp : StripeOp<"terminate", [Terminator]> {
   let printer = [{ *p << getOperationName(); }];
 }
 
+def ReturnOp : StripeOp<"return", [Terminator, HasParent<"ParallelForOp">]> {
+  let summary = "Indicates where a ParallelForOp should execute";
+
+  // 'from' represents the executor value being returned.  We're using a
+  // Variadic here in lieu of Optional; a Stripe Return op may only have zero
+  // or one argument/s.
+  let arguments = (ins Variadic<TensorRefType>:$from);
```

Comment:
Can we pass in an `llvm::None` in the case of no executor being set? Variadic seems too loose.

Reply:
Also, does this replace `TerminateOp`?

Reply:
`llvm::None` doesn't work (`NoneType` can't be used for a `Value*`), and `nullptr` causes a segfault.  I got this style from another MLIR dialect (SPIR-V, I think?); I couldn't find a useful `Optional`.

I think we still want `TerminateOp`, at a minimum for constraints.

Reply:
We could have 2 different ops? What if `TerminateOp` was the no-operand version, and `ReturnOp` was renamed `ExecuteOnOp` that takes one operand? Initially we start with `TerminateOp` from initial tile -> stripe lowering.

Reply:
👍 Done